modules/images/webp-uploads/load.php

```diff
@@ -285,3 +288,47 @@ function webp_uploads_remove_sources_files( $attachment_id ) {
 }
 
 add_action( 'delete_attachment', 'webp_uploads_remove_sources_files', 10, 1 );
+
+/**
+ * Filter on `wp_get_missing_image_subsizes` acting as an action for the logic of the plugin
+ * to determine if additional mime types still need to be created.
+ *
+ * @since n.e.x.t
+ *
+ * @see   wp_get_missing_image_subsizes()
+ *
+ * @param array[] $missing_sizes Associative array of arrays of image sub-size.
+ * @param array   $image_meta    The metadata from the image.
+ * @param int     $attachment_id The ID of the attachment.
+ *
+ * @return array[] $missing_sizes Associative array of arrays of image sub-size.
+ */
+function webp_uploads_wp_get_missing_image_subsizes( $missing_sizes, $image_meta, $attachment_id ) {
+	$trace = array();
+	// Only setup the trace array if we no longer have more sizes.
+	if ( empty( $missing_sizes ) ) {
```

Comment:
No need to define the `$trace` variable here, it's only needed when we also use the `debug_backtrace()` workaround. Let's also reverse the condition here (similar to my other comments above) to use an early return instead, to avoid the nesting, which makes the code easier to read.
```php
if ( ! empty( $missing_sizes ) ) {
	return $missing_sizes;
}
// Only run all custom logic here, without nesting.
```